redwood-cli/src/test/kotlin/app/cash/redwood/generator/ComposeGenerationTest.kt

```diff
@@ -56,10 +56,7 @@ class ComposeGenerationTest {
     val fileSpec = generateComposable(schema, schema.widgets.single())
     assertThat(fileSpec.toString()).apply {
       contains("scoped: @Composable @ScopedAndUnscopedSchemaComposable RowScope.() -> Unit")
-      contains("RowScope.scoped()")
```

Comment:
Removed these because they're testing implementation details. You can't invoke this lambda without a valid receiver so there's no point testing that we emitted code which does so.